src/plugins/explore/public/components/experience_banners/new_experience_banner/new_experience_banner.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+import './new_experience_banner.scss';
+import { NEW_DISCOVER_INFO_URL, SHOW_CLASSIC_DISCOVER_LOCAL_STORAGE_KEY } from '../constants';
+
+export const NewExperienceBanner = () => {
+  const [isVisible, setIsVisible] = useState<boolean>(true);
+
+  // TODO: This should open a modal to add more friction
```

Comment:
i don't really think a modal would add friction if user has decided to switch, but maybe we can get some feedback